lib/plugin/routes/public/users.js

```diff
@@ -306,6 +306,50 @@ exports.register = function (server, options, next) {
     }
   })
 
+  server.route({
+    method: 'GET',
+    path: '/authorization/users/{id}/teams',
+    handler: function (request, reply) {
+      const { id } = request.params
+      const { organizationId } = request.udaru
+      const limit = request.query.limit || server.udaruConfig.get('authorization.defaultPageSize')
```

Comment:
I see some duplication here.

Reply:
this protection will be done in core, I will remove it from routes. Seems that many routes have it.